samples/serverReports/typings/sqlops.d.ts

```diff
@@ -0,0 +1,1530 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+declare module 'sqlops' {
```

Comment:
can't we just reference the sqlops.d.ts with a relative path like we do with other extensions to avoid copying the code?

Reply:
I am no sure. @anthonydresser any idea?

---
In reply to: [176588147](https://github.com/Microsoft/sqlopsstudio/pull/960#discussion_r176588147) [](ancestors = 176588147)

Reply:
A couple notes. 

Simple answer is yes. 

But the non-simple answer is we probably don't want to do that since these are samples, therefore I'd imagine people will just copy paste these. So I would say don't copy paste it here, but refer to the sqlops node module, so that if people copy paste it will still work for them since we aren't directly referring to it relatively.

Reply:
do we have  sqlops node module? 

---
In reply to: [176602389](https://github.com/Microsoft/sqlopsstudio/pull/960#discussion_r176602389) [](ancestors = 176602389)

Reply:
`"sqlops": "github:anthonydresser/vscode-extension-vscode"`